<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LP-214844616" failures="1" tests="1" name="Default test" time="34.501" errors="0" timestamp="2023-07-10T16:55:43 IST">
  <testcase classname="org.hcl.tests.MyntraTest" name="testSoftAssertion" time="24.139">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	url is not Matching expected [https://www.myntra.com/mens-watches?f=Categories%3AFitness%20Bands&amp;amp;sort=recommended] but found [https://www.myntra.com/mens-watches?f=Categories%3AFitness%20Bands%3A%3AColor%3ABlack_36454f&amp;amp;sort=discount],
	price is not Matching expected [₹2399] but found [₹1399]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
url is not Matching expected [https://www.myntra.com/mens-watches?f=Categories%3AFitness%20Bands&sort=recommended] but found [https://www.myntra.com/mens-watches?f=Categories%3AFitness%20Bands%3A%3AColor%3ABlack_36454f&sort=discount],
price is not Matching expected [₹2399] but found [₹1399]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at org.hcl.tests.MyntraTest.testSoftAssertion(MyntraTest.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testSoftAssertion -->
  <testcase classname="org.hcl.tests.MyntraTest" name="testMyntra" time="0.0">
    <ignored/>
  </testcase> <!-- testMyntra -->
</testsuite> <!-- Default test -->
